src/pages/commits/InactiveCommits/index.tsx

```diff
@@ -192,12 +191,9 @@ const InactiveCommitLine = styled.div`
   border: 1px dashed ${gray.light1};
 `;
 
-const TooltipContainer = styled.div`
+// @ts-expect-error
+const StyledTooltip = styled(Tooltip)`
```

Comment:
It seems like the `(see more commits...)` line got uncentered

<img width="299" alt="Screen Shot 2022-12-05 at 2 24 59 PM" src="https://user-images.githubusercontent.com/47064971/205725511-13ca1ee8-9e7e-4cee-8171-4354f4d89334.png">